homeassistant/components/update/strings.json

```diff
@@ -79,5 +92,17 @@
       "name": "Skip update"
     }
   },
-  "title": "Update"
+  "title": "Update",
+  "triggers": {
+    "update_become_available": {
```

Comment:
Please update the trigger names and grammar to follow our previous patterns. Take care since the different translation strings should use different tense in the different cases, key, description and name.

Example:
https://github.com/home-assistant/core/blob/78af3acf355e211fedebdfab0687fcab88b5bc07/homeassistant/components/light/strings.json#L512-L520